modules/agar/src/test/ts/atomic/api/TestStoreTest.ts

```diff
@@ -0,0 +1,125 @@
+import { UnitTest } from '@ephox/bedrock-client';
+import { Arr } from '@ephox/katamari';
+
+import * as GeneralSteps from 'ephox/agar/api/GeneralSteps';
+import { Pipeline } from 'ephox/agar/api/Pipeline';
+import { Step } from 'ephox/agar/api/Step';
+import { TestStore } from 'ephox/agar/api/TestStore';
+
+interface TestData {
+  name: string;
+  age: number;
+  isHappy: boolean;
+}
+
+UnitTest.asyncTest('TestStoreTest strings', (success, failure) => {
```

Comment:
New tests should be using `describe` and `it` BDD style instead of the deprecated/legacy `UnitTest` module.

- [ ] Swap to using BDD style testing

Reply:
Sorry, I copied the other tests in agar which is using the legacy `UnitTest` module. Have converted them to using `describe` and `it`.